support/docker/production/config/production.yaml

```diff
@@ -45,3 +45,9 @@ log:
 
 admin:
   email: null
+
+transcoding:
+  resolutions:
```

Comment:
Could you remove it? I'm not sure we want to transcode to multiple resolutions by default (or at least it should be consistent with the main production.yaml file).